CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing to the BCI-dockerfile-generator
+
+##  Opening an Pull Request
+
+The BCI-dockerfile-generator documentation has a section on contributing
+a new container or modifying an existing container. Please check the
+chapter **Adding and modifying containers** for details.
+
+Before creating a pull request, please format your source code with
+ruff, which is installed in the poetry venv by default:
```

Comment:
nit: I would write Ruff, Poetry, and virtual environment because we are referring to tools, not commands.

Reply:
done